site/en/docs/extensions/reference/declarativeNetRequest/index.md

```diff
@@ -45,8 +45,10 @@ of type [Ruleset][4], as shown below.
 ## Rule Resources
 
 An extension can specify up to [MAX_NUMBER_OF_STATIC_RULESETS][5] [rulesets][6] as part of the
-`"rule_resources"` manifest key. An extension is allowed to enable at least
-[GUARANTEED_MINIMUM_STATIC_RULES][7] static rules. Additional static rule sets may or may not be
+`"rule_resources"` manifest key. An extension is allowed to enable up to
+[MAX_NUMBER_OF_ENABLED_STATIC_RULESETS][18] of those rulesets at any one time, assuming static rule
+limits are not exceeded. An extension is allowed to enable at least
```

Comment:
nit: Change to:

Only [MAX_NUMBER_OF_ENABLED_STATIC_RULESETS][18] of these rulesets can be enabled at a time, assuming static rule limits are not exceeded.

Reply:
nit: linebreak before this.

Reply:
Done.